uaclient/entitlements/esm.py

```diff
@@ -38,3 +38,8 @@ def repo_pin_priority(self) -> "Optional[str]":
     def disable_apt_auth_only(self) -> bool:
         """Only trusty esm-infra should remove apt auth files upon disable"""
         return bool("trusty" == util.get_platform_info()["series"])
+
+    @property
+    def help_info(self) -> str:
+        """Help information for the entitlement"""
+        return "esm-infra help\nInformation about esm-infra"
```

Comment:
Are we going to move the content to a non-python file as a followup then? I expect this content to be multiple paragraphs, with links, etc and making it more trivial to update is definitely the goal.

Reply:
II though the idea was to do something like the `machine_token` property, where we parse a json file and obtain the information that we need:

https://github.com/canonical/ubuntu-advantage-client/blob/master/uaclient/config.py#L134

Since we still do not have the source of that information or the format that it will come, I have not made any assumptions on that part of the code. But, if you think there are some assumptions that we can code right now, just let me know and I can do that without an issue

Reply:
I think asking lech to write out json is a bit much. I was hoping either YAML or individual text files perhaps? 

Maybe we can have a help.py that's just a simple dict like:

    help['esm-infra'] = """
    <Lech to fill all this out here>
    """
    help['esm-apps'] = """
    <lech to fill out>
    """

Reply:
I think both ideas work, I would prefer to have an YAML, but that may cause the problem that if we update that YAML and break it, we would make all of the service's help command to fail. But we can discuss that.

But yes, I can create a help.py file with that dummy info for now and read it from there

Reply:
@lucasmoura @mitechie if we don't like JSON for non-programming types, I'd really suggest that writing a python dict is likely an equivalent barrier to entry for these strings. Let's go YAML if we really don't like JSON.

Reply:
@lucasmoura since we really shouldn't be updating this yaml on the fly at the moment, I don't expect us to be too concerned with breaking help. But, maybe it does make sense to return a "no help available" string if we struggle to correctly parse the help.yaml file. I *think* that file should live in /var/lib/ubuntu-advantage/help.yaml. maybe in /etc/ubuntu-advantage (but filesystem heirarchy standard says that /etc is for "configuration" files, which this is not really).